src/lib/protocols/syslog.c

```diff
@@ -107,7 +108,7 @@ void init_syslog_dissector(struct ndpi_detection_module_struct *ndpi_struct, u_i
   ndpi_set_bitmask_protocol_detection("Syslog", ndpi_struct, detection_bitmask, *id,
 				      NDPI_PROTOCOL_SYSLOG,
 				      ndpi_search_syslog,
-				      NDPI_SELECTION_BITMASK_PROTOCOL_V4_V6_TCP_OR_UDP_WITH_PAYLOAD_WITHOUT_RETRANSMISSION,
+				      NDPI_SELECTION_BITMASK_PROTOCOL_V4_V6_UDP_WITH_PAYLOAD,
```

Comment:
In theory, syslog over TCP is possible. But in practice, I've never seen such aliens.

Reply:
According to https://twitter.com/webernetz/status/1401883154298908673 it shoudn't be so rare and that is a pcap example [syslog_over_tcp.zip](https://github.com/ntop/nDPI/files/8839546/syslog_over_tcp.zip)
Anyway, I agree with your proposal: remove TCP support and let's fight this battle another day, only if really needed.

Reply:
I will revert to UDP&TCP detection and try something else.

Reply:
As you like. I am fine with both solutions (Syslog TCP+UDP or only UDP)